### ISSUE TITLE
Hi,I have a question about the loss?

### ISSUE THREAD

**Issue by BCWang93** [wkentaro/pytorch-fcn#119]
this is the result of my training result,I found the loss is always a number.like this:
'
poch:[2/50], Step[490/1464],Loss:3.0447
Epoch:[2/50], Step[500/1464],Loss:3.0447
Epoch:[2/50], Step[510/1464],Loss:3.0447
Epoch:[2/50], Step[520/1464],Loss:3.0447
Epoch:[2/50], Step[530/1464],Loss:3.0447
Epoch:[2/50], Step[540/1464],Loss:3.0447
Epoch:[2/50], Step[550/1464],Loss:3.0447
Epoch:[2/50], Step[560/1464],Loss:3.0447
Epoch:[2/50], Step[570/1464],Loss:3.0447
Epoch:[2/50], Step[580/1464],Loss:3.0447
Epoch:[2/50], Step[590/1464],Loss:3.0447
Epoch:[2/50], Step[600/1464],Loss:3.0447
Epoch:[2/50], Step[610/1464],Loss:3.0447
Epoch:[2/50], Step[620/1464],Loss:3.0447
Epoch:[2/50], Step[630/1464],Loss:3.0447
Epoch:[2/50], Step[640/1464],Loss:3.0447
Epoch:[2/50], Step[650/1464],Loss:3.0447
Epoch:[2/50], Step[660/1464],Loss:3.0447
Epoch:[2/50], Step[670/1464],Loss:3.0446
Epoch:[2/50], Step[680/1464],Loss:3.0447
Epoch:[2/50], Step[690/1464],Loss:3.0447
Epoch:[2/50], Step[700/1464],Loss:3.0445
Epoch:[2/50], Step[710/1464],Loss:3.0447
Epoch:[2/50], Step[720/1464],Loss:3.0447
Epoch:[2/50], Step[730/1464],Loss:3.0447
Epoch:[2/50], Step[740/1464],Loss:3.0447
Epoch:[2/50], Step[750/1464],Loss:3.0447
Epoch:[2/50], Step[760/1464],Loss:3.0446
Epoch:[2/50], Step[770/1464],Loss:3.0447
Epoch:[2/50], Step[780/1464],Loss:3.0447
Epoch:[2/50], Step[790/1464],Loss:3.0447
Epoch:[2/50], Step[800/1464],Loss:3.0447
Epoch:[2/50], Step[810/1464],Loss:3.0447
Epoch:[2/50], Step[820/1464],Loss:3.0447
Epoch:[2/50], Step[830/1464],Loss:3.0447
Epoch:[2/50], Step[840/1464],Loss:3.0447
Epoch:[2/50], Step[850/1464],Loss:3.0447
Epoch:[2/50], Step[860/1464],Loss:3.0447
Epoch:[2/50], Step[870/1464],Loss:3.0447
Epoch:[2/50], Step[880/1464],Loss:3.0447
Epoch:[2/50], Step[890/1464],Loss:3.0447
Epoch:[2/50], Step[900/1464],Loss:3.0447
Epoch:[2/50], Step[910/1464],Loss:3.0447
Epoch:[2/50], Step[920/1464],Loss:3.0447
Epoch:[2/50], Step[930/1464],Loss:3.0447
Epoch:[2/50], Step[940/1464],Loss:3.0447
Epoch:[2/50], Step[950/1464],Loss:3.0447
Epoch:[2/50], Step[960/1464],Loss:3.0447
Epoch:[2/50], Step[970/1464],Loss:3.0447
Epoch:[2/50], Step[980/1464],Loss:3.0445
Epoch:[2/50], Step[990/1464],Loss:3.0446
Epoch:[2/50], Step[1000/1464],Loss:3.0447
Epoch:[2/50], Step[1010/1464],Loss:3.0444
Epoch:[2/50], Step[1020/1464],Loss:3.0447
Epoch:[2/50], Step[1030/1464],Loss:3.0447
Epoch:[2/50], Step[1040/1464],Loss:3.0447
Epoch:[2/50], Step[1050/1464],Loss:3.0447
Epoch:[2/50], Step[1060/1464],Loss:3.0446
Epoch:[2/50], Step[1070/1464],Loss:3.0447
Epoch:[2/50], Step[1080/1464],Loss:3.0447
Epoch:[2/50], Step[1090/1464],Loss:3.0447
'
whether I have some wrong?Or,there have some problem?Thanks!
